### PR TITLE
Add vhd-format{,-lwt}.0.9.1

### PR DESCRIPTION
**Pull request by djs55**
Signed-off-by: David Scott <dave@recoil.org>

### CODE REVIEW

**Comment by mseri** [comment]
The failures appear to be the expected ones: unsupported compiler version

**Comment by mseri** [comment]
Can you please update the minimal supported compiler of the lwt library to `4.03.0` as 
```
vhd-format-lwt -> mirage-types-lwt >= 3.0.0 -> mirage-types >= 3.0.0 -> mirage-stack >= 1.0.0 -> mirage-protocols >= 1.0.0
```

Also, `vhd-format` is failing with
```
 File "vhd_format/f.mli", line 124, characters 31-46:
- Error: Unbound type constructor result
```
I am not sure if it's worth to add the `ocaml-result` dependency to make it compatible to `4.02.3` when porbably there is some dependency that has anyways dropped support for it

**Comment by mseri** [comment]
If you drop support for 4.02.3 on the opam files, I am happy to merge this

File: packages/vhd-format-lwt/vhd-format-lwt.0.9.1/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "org:xapi-project"
+]
+authors: [ "Dave Scott" "Jon Ludlam" ]
+homepage: "https://github.com/mirage/ocaml-vhd"
+bug-reports: "https://github.com/mirage/ocaml-vhd/issues"
+dev-repo: "git://github.com/mirage/ocaml-vhd"
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
+build-test: [["jbuilder" "runtest" "-p" name "-j" jobs]]
+depends: [
+  "cstruct"
```
Comment (avsm) [inline packages/vhd-format-lwt/vhd-format-lwt.0.9.1/opam]: missing a jbuilder dependency here

Comment (mseri) [reply packages/vhd-format-lwt/vhd-format-lwt.0.9.1/opam]: Yes, also in `vhd-format.opam`

**Comment by djs55** [comment]
- bumped min OCaml version to 4.03.0
- added `jbuilder {build}`

**Comment by avsm** [comment]
Thanks! You may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Pull request merged**